### PR TITLE
Rename devpage.html file

### LINKED ISSUE #1321: Rename devpage.html
**Expected behavior**
Rename devpage.html to engineering-team.html

**Are you working on it?**
Yes

### PR DESCRIPTION
**Pull request by RandilaP**
## Purpose

The purpose of this PR is to fix #1321

### CODE REVIEW

**Comment by RandilaP** [comment]
@jayasanka-sack @anjula-sack Can u merge this

**Review by Piumal1999 — COMMENTED** [review]
@RandilaP Let's remove the devpage.html file.

**Comment by anjula-sack** [comment]
@RandilaP can you resolve the conflict?

**Comment by RandilaP** [comment]
@anjula-sack Sure.

**Comment by anjula-sack** [comment]
@RandilaP Can you create an issue for this? You should always send a PR if there's an issue for that. If not you can create it.

**Comment by RandilaP** [comment]
@anjula-sack Okay

**Comment by Piumal1999** [comment]
Hi @RandilaP, any update on this?

**Comment by YoshithaRathnayake** [comment]
> Hi @RandilaP, any update on this?

Shall I create an issue and do this @Piumal1999 and @anjula-sack ?

**Comment by Piumal1999** [comment]
@RandilaP This Pull request contains some unwanted changes. This should only rename the HTML file and update the links

**Comment by anjula-sack** [comment]
@RandilaP can you create an issue for this an link it?

File: engineering-team.html [hunk 1]
```diff
@@ -11,7 +11,7 @@
         </script>
         <meta charset="utf-8">
         <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
-        <meta name="description" content="The engineering team is the backbone of the SEF community. This team is saturated with potent young talent who engage in various challenges when it comes to contribution.">
```
Comment (anjula-sack) [inline engineering-team.html]: This shouldn't be removed

File: engineering-team.html [hunk 2]
```diff
@@ -44,7 +44,7 @@
                     <div class="pl-4">
                        <h1 class="text-roboto text-capitalize pb-2">SEF Engineering Team</h1>
                         <p>Being part of the SEF engineering team is a lot more than making a pull request. You get first-hand experience in handling the latest cutting-edge technologies and it will be required of you to develop skills that are essential to being part of the modern industry. With our community being active through various mediums such as standup meetings, knowledge sharing sessions and communication platform groups, you will soon be exposed to the startup culture, the various practices and procedures involved with contributing to projects.  </p>
-                            <a href="/join-us.html">
+                            <a target="_blank"  href="/join-us.html">
```
Comment (anjula-sack) [inline engineering-team.html]: I don't think we need to open the join us page in a new tab.

File: engineering-team.html [hunk 3]
```diff
@@ -73,13 +73,19 @@ <h3 class="mb-3">ScholarX</h3>
                         
                         <p class="text-justify">"ScholarX is an exclusive program aimed at providing mentoring support for talented undergraduate students in Sri Lanka. Our mentors are engaged with some of the world’s top universities and Fortune 500 companies and are carefully selected for their expertise in academia and industry.
                             This free premium mentoring platform was created for Sri Lankans who see the opportunity and positive change brought about by a culture of knowledge and expertise sharing without the limitations of geographical borders."</p>
-
-                            <a target="_blank" href="https://github.com/sef-global/scholarx" class="btn btn-default mb-2">
+                            <div class="row">
+                            
+                            <div class="col-md-5">
+                            <a target="_blank" href="https://github.com/sef-global/scholarx" class="btn btn-default">
```
Comment (anjula-sack) [inline engineering-team.html]: Undo these changes. As @Piumal1999 says you should only change the file name

**Review by anjula-sack — CHANGES_REQUESTED** [review]

File: engineering-team.html (hunk 3 above)
Comment (RandilaP) [reply engineering-team.html]: @anjula-sack @Piumal1999 I align some buttons as some improvement. I update the PR.

**Comment by anjula-sack** [comment]
@RandilaP Can you resolve the conflicts?

**Comment by RandilaP** [comment]
@anjula-sack I tried to do that. But i couldn't do that.

**Comment by Piumal1999** [comment]
@RandilaP It seems like there are a lot of conflicts. If it is very hard to resolve those one-by-one, you can just hard reset the branch and redo the changes. That would be easier.

```
git checkout dev1
git reset --hard upstream/master
git pull upstream master
// do the changes and commit
git push origin dev1 --force
```

**Comment by RandilaP** [comment]
@anjula-sack @Piumal1999 @jayasanka-sack Is this okay

**Comment by RandilaP** [comment]
@Piumal1999 Thanks for the help.

**Review by anjula-sack — APPROVED** [review]
Thank you! @RandilaP
